operator/deploy/operator-psp.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kiali-operator-psp
```

by convention, all operator entities (such as the Deployment, ClusterRole, ServiceAccount, etc) are named "kiali-operator". So change this to follow that convention:

```suggestion
  name: kiali-operator
```

---

Note that by convention the names of the files here are not prefixed with "operator-" (so you do not see things like "operator-role.yaml" or "operator-namespace.yaml" - its just "role.yaml" and "namespace.yaml".

So rename this file from operator-psp.yaml to simply `psp.yaml`

---

Thanks! I will change the names convention.